sdk-docs/save-data-from-your-app/changing-custom-field-definitions.md

```diff
@@ -43,4 +43,4 @@ Each field type has special rules:
 	    
 	-   The existing value will be saved for retrieval in the API.
 
-	- Regex patterns cannot be changed. 
\ No newline at end of file
+	- Regex patterns can be changed, but be warned that existing data can become uncompliant with the new pattern.
```

Comment:
What happens then? Will the data be lost, or will it still be visible in the UI but not editable?

Reply:
We need to talk with @ext-kmu or @jbd-trackunit to confirm the behavior. Will you do that in another PR @osmantoplica-trackunit ?

Reply:
Very good question..

Reply:
I'll park it as a TODO on my Trello board and elaborate in a different PR.
There was some clarification here: https://trackunit.slack.com/archives/C031USWAFL1/p1681800585220629

But there are still open-ended questions to what it means in practice.